res/values/attrs.xml

```diff
@@ -108,6 +108,7 @@
     <attr name="menu_save_icon" format="reference" />
     <attr name="menu_pin_icon" format="reference" />
     <attr name="menu_unpin_icon" format="reference" />
+    <attr name="menu_emoji" format="reference" />
 
```

Comment:
Extra line

Reply:
@Radu-Raicea  I checked, this space is acceptable